docs/administration/configuration/configuring-interactive-sessions/index.md

```diff
@@ -0,0 +1,53 @@
+# Configuring interactive sessions
+
+## Interactive sessions cleanup
+
+### Automatic and scheduled closure
+
+By default, interactive sessions in REANA are kept open indefinitely. However, administrators
+have the flexibility to configure the maximum inactivity period after which interactive
+sessions will be automatically closed; the schedule of this cleanup process can be
+customized. This helps optimize resource utilization and ensure that unused sessions
+are not consuming unnecessary resources.
+
+The automatic closure of interactive sessions in REANA can be customized by configuring
+[Helm values](https://github.com/reanahub/reana/tree/master/helm/reana).
+The following Helm values allow you to control the behavior of the automatic closure feature:
+
+- `interactive_sessions.maximum_inactivity_period`:
+  This option allows you to set a limit (in days) for the maximum inactivity period of interactive sessions after which they will be closed.
+
+- `interactive_sessions.cronjob_schedule`:
+  This option allows you to set the schedule for the cleanup process using the cronjob format.
+
+Example:
+
+```yaml
+interactive_sessions:
+    maximum_inactivity_period: 7
+    cronjob_schedule: "0 3 * * *"
+...
```

Comment:
What do you think about removing `...` and adding the copy-to-clipboard button to this snippet? Example: https://github.com/reanahub/docs.reana.io/blob/4149b8d8fa4b219a0ed4fdf74d5e4719fbe8b061/docs/administration/configuration/configuring-storage-volumes/index.md?plain=1#L40

Reply:
The copy-to-clipboard button is not being shown, probably because there is a missing `.` before `yaml`